client/src/partials/home/home.html

```diff
@@ -1,44 +1,128 @@
 <div class="flex-header">
   <div class="bhima-title">
-
     <ol class="headercrumb">
-      <li class="static">{{ "HOME.TITLE" | translate }}</li>
-      <li class="title">{{HomeCtrl.enterprise.name}}</span></li>
+      <li class="static">{{ ::HomeCtrl.enterprise.name }}</li>
+      <li class="title">{{ ::HomeCtrl.project.name }}</li>
     </ol>
-
   </div>
 </div>
 
 <div class="flex-content">
   <div class="container-fluid">
-    
-    <div class="page-header">
-      <h1>{{HomeCtrl.enterprise.name}}</h1>
-    </div>
 
-    <div class="panel panel-default">
-      <div class="panel-heading">{{ "HOME.TITLE" | translate }}</div>
-      <div class="panel-body"> 
-        <div class="row">
-          <div class="col-sm-3">
-            {{ "HOME.PROJECT_NAME" | translate }}:
-            <br><strong>{{ HomeCtrl.project.name }}</strong>
+    <div class="row">
+
+      <!-- project information panel -->
+      <div class="col-md-8">
+        <div class="panel panel-default" style="min-height:30vh;">
+          <div class="panel-body">
+            <h4>
+              {{ ::HomeCtrl.project.name }} <small>{{ ::HomeCtrl.project.abbr }}</small>
+              <small class="pull-right"><span class="glyphicon glyphicon-calendar"></span> {{ ::HomeCtrl.today | date }}</small>
+            </h4>
+            <dl>
+              <dt>{{ ::"FORM.LABELS.CURRENCY" | translate }}</dt>
+              <dd>{{ ::HomeCtrl.enterprise.currencyLabel }}</dd>
+
+              <dt>{{ ::"TABLE.COLUMNS.EXCHANGE_RATE" | translate }}</dt>
+              <dd>
+                <div ng-repeat="currency in HomeCtrl.currencies track by currency.id" style="padding-left:15px;">
+                  <em>{{ currency.name }}</em> {{ currency.rate | currency:currency.id }}
+                  <a href ui-sref="exchange">
+                    <span class="glyphicon glyphicon-link"></span>
+                    {{ ::"FORM.BUTTONS.UPDATE" | translate }}
+                  </a>
+                </div>
+              </dd>
+
+              <dt>
+                {{ ::"TREE.FISCAL_YEAR" | translate }}
+                <a href>
+                  <span class="glyphicon glyphicon-info-sign text-info"></span>
+                </a>
+              </dt>
+              <dd>
+                <div class="progress">
+                  <div class="progress-bar" role="progressbar" style="width: 20%;">
+                    30%
```

Comment:
Yes, complete POC.  I'll add in a comment.  FWIW, I mentioned it in the PR, but future devs will be reading the code, not the PR comments.

Good suggestion.